src/scss/styles.scss

```diff
@@ -1,20 +1,51 @@
 @import "bourbon";
 @import "neat";
 
+@import url(http://fonts.googleapis.com/css?family=Raleway:400,600,700,300);
+@import url(http://fonts.googleapis.com/css?family=Roboto:400,600,700,300);
+

```

Comment:
Cool. I am like 87% sure it was my first pull request haha. I have seen
some projects that use the material palette and then $brand-variable naming
convention that seem to organize things well.

On Fri, Mar 4, 2016 at 1:45 AM, Zack Hall <notifications@github.com> wrote:

> In src/scss/styles.scss
> <https://github.com/zackhall/chug/pull/2#discussion_r54995502>:
>
> > @@ -1,20 +1,51 @@
> >  @import "bourbon";
> >  @import "neat";
> >
> > +@import url(http://fonts.googleapis.com/css?family=Raleway:400,600,700,300);
> > +@import url(http://fonts.googleapis.com/css?family=Roboto:400,600,700,300);
> > +
>
> Gotcha, gotcha. I can't really pull this as is, but if you want to clean
> up the .scss file to make the minimal changes you'd like to see in the
> master repo and add another commit I'd be happy to pull it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zackhall/chug/pull/2/files#r54995502>.
>



-- 

Michael Vieth